patches/npm/@openeuropa+bcl-theme-default+0.24.1.patch

```diff
@@ -11,6 +11,50 @@ index 6e08be9..ddc41ad 100644
        {% set _caption_classes = 'carousel-caption' %}
        {% if _item.caption is not empty and _item.link is empty %}
          {% set _caption_classes = _caption_classes ~ ' d-none d-md-block' %}
+diff --git a/node_modules/@openeuropa/bcl-theme-default/templates/bcl-link/link.html.twig b/node_modules/@openeuropa/bcl-theme-default/templates/bcl-link/link.html.twig
+index 0ccc7ec..97082d8 100644
+--- a/node_modules/@openeuropa/bcl-theme-default/templates/bcl-link/link.html.twig
++++ b/node_modules/@openeuropa/bcl-theme-default/templates/bcl-link/link.html.twig
+@@ -1,4 +1,4 @@
+-{% spaceless %}
++{% apply spaceless %}
+ 
+ {# Parameters:
+   - label (string) (default: '')
+@@ -72,18 +72,20 @@
+ {% if _classes is not empty %}
+   {% set attributes = attributes.addClass(_classes) %}
+ {% endif %}
+-{% set attributes = attributes.setAttribute('href', _path) %}
+ 
+-<a
+-  {{ attributes }}
+->
+-  {%- if _icon_position == 'before' and _icon_markup is defined -%}
+-    {{ _icon_markup|raw }}
+-  {%- endif -%}
+-  {{- _label -}}
+-  {%- if _icon_position == 'after' and _icon_markup is defined -%}
+-    {{ _icon_markup|raw }}
+-  {%- endif -%}
+-</a>
++{% if _icon_markup is defined %}
++  {%- set _label -%}
++    {%- if _icon_position == 'before'  -%}
++      {{ _icon_markup|raw }}
```

Comment:
Is the |raw needed? I tried locally and it seems it wasn't. When we set rendered html to a variable in a twig file, it gets set as safe markup.